packages/@okta/vuepress-site/docs/concepts/third-party-cookies/index.md

```diff
@@ -0,0 +1,155 @@
+
+For example, if your original Okta org is `companyname.okta.com`, and your app server is `app.companyname.com`, you would use the custom URL domain feature to give your Okta org a new URL like `login.companyname.com`. This puts your app and your Okta org within the same site.
+
+It's okay for the Okta org and the app server to have different subdomains (in the above example, `app` and `auth`). Only the effective Top-Level Domain plus the one label immediately to its left (eTLD+1) need to be the same (in the example, `companyname.com`).
+
+After setting up custom URL domain, your need to update your configuration of the Okta Sign-In Widget and of any Okta SDKs, so that they use your new custom domain as the base URL of your org.
+
+If your code makes any XHR calls directly to Okta endpoints, you need to update the URLs of those calls.
+
+The following sections show examples of these changes. The examples assume your original Okta Org URL is `companyname.okta.com` and your custom URL domain is `login.companyname.com`.
+
+### Sign-In Widget
+
+For information on updating the configuration of the Okta Sign-In Widget, see <https://github.com/okta/okta-signin-widget#basic-config-options>
+
+Example change:
+
+```
+ var signIn = new OktaSignIn({
+-  baseUrl: 'https://companyname.okta.com'
++  baseUrl: 'https://login.companyname.com'
+ });
+```
+
+### Auth JavaScript
+
+For information on updating the configuration of the Okta Auth JavaScript SDK, see <https://github.com/okta/okta-auth-js/blob/master/README.md>.
+
+Example change:
+
+```
+ var authClient = new OktaAuth({
+-  issuer: 'https://companyname.okta.com'
++  issuer: 'https://login.companyname.com'
+ });
+```
+
+### JavaScript React
+
+For information on updating the configuration of a React application, see <https://github.com/okta/samples-js-react#readme>.
+
+Example change:
+
+```
+import { SecureRoute, Security, LoginCallback } from '@okta/okta-react';
+
+class App extends Component {
+   render() {
+     return (
+       <Router>
+-        <Security issuer='https://companyname.okta.com/oauth2/default'
++        <Security issuer='https://login.companyname.com/oauth2/default'
+                   clientId='{clientId}'
+                   redirectUri={window.location.origin + '/implicit/callback'} >
+           <Route path='/' exact={true} component={Home}/>
+          <SecureRoute path='/protected' component={Protected}/>
+          <Route path='/implicit/callback' component={LoginCallback} />
+        </Security>
+      </Router>
+    );
+  }
+}
+
+```
+
+###  JavaScript Angular
+
+For information on updating the configuration of an Angular application, see <https://github.com/okta/samples-js-angular#readme>.
```

Comment:
ah, I see where you from now. You're looking at sample project but I'm looking at SDK itself. https://github.com/okta/okta-oidc-js/tree/master/packages/okta-angular

Reply:
Do we have a standard for what we refer to in situations like this? Seems like the SDK would be the right developer tool to refer to?

Reply:
Ping @robertdamphousse-okta @aarongranick-okta .
My intuition was SDK doc as well.

Reply:
Yes, lets refer to the SDK

Reply:
Changed all links to point to SDK docs.